virttest/libvirt_storage.py

```diff
@@ -544,3 +544,17 @@ def clone_volume(self, old_name, new_name):
             logging.info("Volume '%s' does not exist or '%s' has been exist."
                          % (old_name, new_name))
             return False
+
+
+def check_qemu_image_locking_feature_support():
+    """
+    QEMU commit 244a566(qemu-2.10) introduced the image locking feature which brought
+    a new option '-U' to the qemu-img command:info, compare, check, bench, convert,
+    dd, map, snapshot, rebase. This method provides one way to determine whether
+    current qemu-img support or not.
+
+    :return: True if current qemu-img command support
+    """
+    cmd_result = process.run("qemu-img" + ' -h', ignore_status=True,
+                             shell=True, verbose=False)
+    return '-U' in cmd_result.stdout
```

Comment:
we can re-use `class QemuImg(storage.QemuImg)` in virttest/qemu_storage.py

Reply:
@balamuruhans , I observed some change introduced in qemu_storage.py.
But actually in our libvirt test code (tp-libvrit), it  rarely use this module to create or info image operations,but hard code qemu-img command. 
This patch was used to provide one solution for those test case,which use qemu-img commands directly.

Reply:
@chunfuwen okay, agreed